.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -323,6 +323,22 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refTime)
                 }
             }
 
+            /* Taskmode specific modification of past datetime references
+               Example: 22 april at 5 pm. (reference time is 22/04/2022 T17:30:00)
+               Modification under taskmode.

```

Comment:
This comment could be clearer on what the difference is. But won't block merge.

Reply:
Added detailed comment.